dask/dataframe/tests/test_arithmetics_reduction.py

```diff
@@ -1288,34 +1288,51 @@ def test_reductions_frame_dtypes_numeric_only_supported():
     )
 
     ddf = dd.from_pandas(df, 3)
-    funcs = ["sum"]
+    funcs = ["sum", "prod", "product", "min", "max"]
```

Comment:
It might be good to use these in `pytest.mark.parametrize`, if possible. There, you could also mark some as `xfail`.

Reply:
Yeah I thought about this as well, modelled after the test below that uses the same pattern. Not sure why that was, performance might be a reason? Happy to change though if you prefer.

One thing about xfails: They really slow down the test suite if you have too many of them.

Reply:
Not sure why the original tests were formatted this way. Not a huge deal either way, but FWIW I also prefer `pytest.mark.parametrize`. Pushed a tiny commit that moves `funcs` into a parameterization. 

> One thing about xfails: They really slow down the test suite if you have too many of them.

Yeah, fair point. We could avoid `pytest.mark.parametrize` in these cases or even just mark them as `skip` instead of `xfail` -- this isn't exactly the same, but may be close enough

Reply:
Yeah I prefer parametrisation as well. Saw the loop logic a couple of times in the test suite, so wasn't sure if this was preferred here.

Yeah skip is better performance wise